demisto_sdk/commands/content_graph/interface/neo4j/queries/common.py

```diff
@@ -54,10 +58,26 @@ def to_neo4j_map(properties: dict) -> str:
 
 
 def run_query(tx: Transaction, query: str, **kwargs) -> Result:
+    result = None
     try:
         start_time: datetime = datetime.now()
         logger.debug(f"Running query:\n{query}")
-        result = tx.run(query, **kwargs)
+        # invoke a new thread and execute `tx.run in a thread. If the query times out, retry.
+        for retry in range(MAX_RETRIES_QUERY):
+            try:
+                with ThreadPoolExecutor(max_workers=1) as executor:
+                    future = executor.submit(tx.run, query, **kwargs)
```

Comment:
please add a log line after the submit. preferably with some id of the transaction.

Reply:
added a log line that the query was executed. The Transaction doesn't have an id and anyway I don't think it is very useful. In the log before we print the actual query that executed, so I think this log provides enough info for us